readthedocs/vcs_support/backends/git.py

```diff
@@ -55,6 +55,10 @@ def repo_exists(self):
         code, _, _ = self.run('git', 'status', record=False)
         return code == 0
 
+    def submodules_exists(self):
+        code, out, _ = self.run('git', 'submodule', 'status', record=False)
+        return code == 0 and bool(out)
```

Comment:
This command always returns 0, but the output is empty when there isn't submodules, I think is better to rely on git than checking for `.gitmodules` file.

Reply:
Well, actually checking for `.gitmodules` file makes the tests more easy to write :/. Please let me know if you want me to go for that path.

Reply:
I figure out

Reply:
I test this locally in a repo with 126 submodules and I found this command is too slow and very CPU consuming:

    git submodule status  0,73s user 0,50s system 100% cpu 1,228 total

Even in the RTD repository:

    $ time git submodule status
    git submodule status  0,08s user 0,05s system 103% cpu 0,129 total

I like following the pattern here by using git commands, but this is something that we will want to consider over just checking the existance of the file.

@agjohnson what do you think?

Reply:
I thought I commented here, but I think i got side tracked determining how `git submodule status` worked, and I think i got stuck in that rabbit hole. Relying on git here is fine. If we find it could be refactored, I still think checking for a top level `.gitmodules` works just as fine also.